src/main/resources/changelog/changes/v3.3/create-reindex-entity-tables.xml

```diff
@@ -0,0 +1,282 @@
+    <preConditions>
+      <not>
+        <indexExists indexName="idx_items_holding_id"/>
+      </not>
+    </preConditions>
+
+    <comment>Create idx_items_holding_id index</comment>
+
+    <createIndex tableName="items" indexName="idx_items_holding_id">
+      <column name="holding_id"/>
+    </createIndex>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-classifications-table" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <tableExists tableName="classifications"/>
+      </not>
+    </preConditions>
+
+    <comment>Create classifications table</comment>
+
+    <createTable tableName="classifications">
+      <column name="classification_number" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_classifications"/>
+      </column>
+      <column name="classification_type_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_classifications"/>
+      </column>
+      <column name="instances_arr" type="jsonb">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-subjects-table" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <tableExists tableName="subjects"/>
+      </not>
+    </preConditions>
+
+    <comment>Create subjects table</comment>
+
+    <createTable tableName="subjects">
+      <column name="subject_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_subjects"/>
+      </column>
+      <column name="subject_value" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_subjects"/>
+      </column>
+      <column name="authority_id" type="VARCHAR(255)" defaultValue="">
+        <constraints nullable="false"/>
+      </column>
+      <column name="instances_arr" type="jsonb">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-contributors-table" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <tableExists tableName="contributors"/>
+      </not>
+    </preConditions>
+
+    <comment>Create contributors table</comment>
+
+    <createTable tableName="contributors">
+      <column name="contributor_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_contributors"/>
+      </column>
+      <column name="contributor_name" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_contributors"/>
+      </column>
+      <column name="authority_id" type="VARCHAR(255)" defaultValue="">
+        <constraints nullable="false"/>
+      </column>
+      <column name="instances_arr" type="jsonb">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
```

Comment:
It should follow same structure as the index: 
```
"name": {
      "index": "keyword_lowercase",
      "showInResponse": [ "browse" ]
    },
    "contributorNameTypeId": {
      "index": "keyword",
      "searchTypes": [ "facet", "filter" ],
      "showInResponse": [ "browse" ]
    },
    "instances": {
      "type": "object",
      "properties": {
        "instanceId": {
          "index": "source"
        },
        "contributorTypeId": {
          "index": "keyword",
          "showInResponse": [ "browse" ]
        },
        "tenantId": {
          "index": "keyword",
          "searchTypes": [ "facet", "filter" ]
        },
        "shared": {
          "index": "bool",
          "searchTypes": [ "facet", "filter" ],
          "default": false
        }
      }
    },
    "authorityId": {
      "index": "keyword",
      "showInResponse": [ "browse" ]
    }
  }
```
`contributorNameTypeId` column is missing. And also please check trigger